app/purchase.tsx

```diff
@@ -0,0 +1,146 @@
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import { StyleSheet, Pressable, Image, View, Text } from 'react-native';
+import { ThemedView } from '@/components/ThemedView';
+import { ThemedText } from '@/components/ThemedText';
+import { Colors } from '@/constants/Colors';
+import Typography from '@/constants/Typography';
+import { s, vs } from 'react-native-size-matters';
+import { Feather } from '@expo/vector-icons';
+
+
+export default function PurchaseScreen() {
+  const { title, price } = useLocalSearchParams();
+  const router = useRouter();
+
+  return (
+    <ThemedView style={styles.container}>
+      <ThemedView style={styles.modalHeader}>
+        <ThemedText style={styles.modalTitle}>결제수단 선택</ThemedText>
+        <Pressable onPress={() => router.back()} style={styles.closeButton}>
+          <Feather name="x" size={s(20)} color="black" />
```

Comment:
X 버튼이 다크모드에 대응되지 않습니다. 다음과 같이 iconColor을 정의하여 색상을 유동적으로 설정 부탁드립니다!

```ts
const theme = useTheme();
const iconColor = getColor(theme, {
    light: Colors.common.black,
    dark: Colors.common.white,
  });
```
참고로 getColor은 현재 앱에서 설정된 화면 모드 (라이트, 다크)를 불러와 반환해주는 함수로 테마에 따라 적절한 색상을 선택해줍니다!
```suggestion
          <Feather name="x" size={s(20)} color="{iconColor}" />
```